common/src/main/java/com/redhat/lightblue/rest/logging/LoggingFilter.java

```diff
@@ -51,9 +55,21 @@ public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain
                 LOGGER.info("ServletResponse of type: " + resp.getClass());
             }
 
+            LOGGER.debug("log for testing");
+
             chain.doFilter(req, resp);
         } finally {
             MDC.remove(HEADER_REQUEST_UUID);
+            MDC.remove(HEADER_REQUEST_PRINCIPAL);
+        }
+    }
+
+    private void principal(HttpServletRequest httpReq) {
```

Comment:
Ugly name. Also, would've been easier to read if this was inline rather than a function I think.